Source/CarthageKit/Project.swift

```diff
@@ -926,6 +913,24 @@ public final class Project {
 					let subdirectoryPath = (CarthageProjectCheckoutsPath as NSString).appendingPathComponent(name)
 					let linkDestinationPath = relativeLinkDestinationForDependencyProject(dependency.project, subdirectory: subdirectoryPath)
 
+					let dependencyCheckoutURLResource = try? dependencyCheckoutURL.resourceValues(forKeys: [
+						.isSymbolicLinkKey,
+						.isDirectoryKey
+					])
+
+					if dependencyCheckoutURLResource?.isSymbolicLink == .some(true) {
+						_ = dependencyCheckoutURL.carthage_path.withCString(Darwin.unlink)
```

Comment:
Yeah, since we're checking let's use `removeItem`. I feel more comfortable with the `NSFileManager` APIs.